platformio.ini

```
@@ -153,13 +153,12 @@ extra_scripts = post:esphome/components/esp32/post_build.py.script
 ; These are common settings for the RP2040 using Arduino.
 [common:rp2040-arduino]
 extends = common:arduino
-board_build.core = earlephilhower
 board_build.filesystem_size = 0.5m
 
-platform = https://github.com/maxgerhardt/platform-raspberrypi.git
+platform = platformio/raspberrypi@1.12.0
```

The published package does not work for ESPHome
```suggestion
platform = https://github.com/maxgerhardt/platform-raspberrypi.git
```